### ISSUE TITLE
UTF-8 document being return as ASCII-8BIT

### ISSUE THREAD

**Issue by ryanswood** [rest-client/rest-client#418]
We're getting this in the headers:

```http
Content-Type:application/json;charset=UTF-8
```

But rest-client is returning

```
"body" => {"encoding"=>"ASCII-8BIT",
```

When we use HTTParty instead, we correctly see:

```
"body"=>{"encoding"=>"US-ASCII"
```

This is particularly bad for us because we're using VCR to replay responses for our test suite, and an ASCII-8BIT response [results in](http://stackoverflow.com/questions/21920259/how-to-edit-response-body-returned-by-vcr-gem) the body being stored in unreadable binary.

Thanks

**Comment by ab** [comment]
What version of rest-client are you using? Until (currently prerelease) 2.0, rest-client did not automatically set String encoding based on headers. So whatever was created by Net::HTTP under the hood would be left unchanged.

**Comment by ryanswood** [comment]
I am using version `1.8.0`

**Comment by mlavandero** [comment]
I am also having the same problem, using `1.8.0` too

**Comment by ab** [comment]
This is a feature, which will be released in `2.0.0`. You can install the pre-release `2.0.0.rc2` if you'd like to use it now. If you want to continue using `1.8.0` you'll have to set the string encoding yourself.
